src/tailwind.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
       },
       colors: {
         inherit: {
-          default: "inherit", // why need this mmm
+          default: "inherit",
```

Comment:
actually do u rmb why we needed inherit? i forgot

Reply:
i think cos there is a case where we need to use inherit but tailwind don't have, so need to extend it